### ISSUE TITLE
lxcfs -d doesn't work

### ISSUE THREAD

**Issue by nehaljwani** [lxc/lxcfs#62]
Readme.md claims that -d can be passed to lxcfs to enter debug mode. But:
```
[root@localhost lxcfs]# git rev-parse HEAD
121013f74ca35633d460a699fb9ddbd46a942632
[root@localhost lxcfs]# lxcfs -s -f -d -o allow_other /usr/local/var/lib/lxcfs
Usage:

lxcfs mountpoint
lxcfs -h
```

**Comment by hallyn** [comment]
Yeah the readme is quite out of date.  Most of the command line options are now ignored.  Ignoring -d was not intentional though.

Patches both to update the readme and to reimplement -d would be happily accepted.

Thanks for point this out.

**Comment by nehaljwani** [comment]
Fixed by 53a866a
